### DISCUSSION TITLE
appium Inspechtor

### DISCUSSION THREAD

**Discussion by khuatvanlong290304** [appium/appium#18847] [General]
hi , i am a beginner in this field , maybe i am wrong from writing this here , but i really dont know where to find help , help me i would like to connect real machine: redmi not 8, but it always current error This error
Can not connect to server; Are you sure it's running? If you are using the browser version, make sure your Appium server is started with --allow-cors. , i tried everything but it still works , please help me , thanks

**Comment by KazuCocoa** [comment]
Perhaps https://discuss.appium.io/ is more helpful to ask general questions
